### PR TITLE
add kylin OS support

### PR DESCRIPTION
**Pull request by shiryu-zhao**
#### Description:

-OpenScap is a well-known security configuration baseline scanning organization in the industry. The tools provided by OpenScap provide security scanning functions for most operating systems around the world. KylinSoft also hopes to join the OpenScap family and use the powerful functions of OpenScap to complete and provide security configuration baseline scanning capabilities.

-KylinSoft is  one of the key operation system vendors in China, and Kylin Linux Advanced Server V10 is one of KylinSoft's important products, which is a robust, secure, and high-performance operating systems tailored to meet the needs of both government and enterprise users. You can learn more about in the website: https://kylinos.cn.

-In the future, the PR on openscap will be maintained and updated by members of our security technology department. 

#### Rationale:

- This PR is to build the kylin-related support framework.

#### Review Hints:

- Please review it and any feedback is my appreciate.

### CODE REVIEW

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR. 

Do you plan on keeping this product up-to-date?

File: products/kylin/product.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+product: kylin
+full_name: Kylin OS
+type: platform
+
+benchmark_id: KYLIN
+benchmark_root: "../../linux_os/guide"
+
+profiles_root: "./profiles"
+
+pkg_manager: "dnf"
```
[the remark this answers is not in the document]
Comment (shiryu-zhao) [reply products/kylin/product.yml:10]: Firstly, thanks very much for your help and concern for the pr that I submitted, I will fix the issue that you mentioned above. 
Secondly, Kylin V10 is a RPM based distro, but I can not find any conflicting information.so I want to ask you for help ,please give me some tips on fixing this problem.Thank you very much!

[the remark this answers is not in the document]
Comment (Mab879) [reply products/kylin/product.yml:10]: Your confirmation is all I need.

**Review by dodys — CHANGES_REQUESTED** [review]
@shiryu-zhao do not merge master into your development branch, do a rebase instead

**Pull request closed without merging**